docs/pages/2019_IIT_Mandi_India/sections/setup.rst

```diff
@@ -157,16 +152,17 @@ In particular, when running the tutorial on a Linux VM, copy the link address an
 
 .. code:: bash
 
-   wget <LINK>
+   wget <URL>
 
-where you replace ``<LINK>`` with the actual HTTPS link copied from the tutorial text in your browser.
-This will download the file in your current directory.
+where you replace ``<URL>`` with the actual HTTPS url copied from the tutorial text in your browser.
```

Comment:
```suggestion
where you replace ``<URL>`` with the actual HTTPS URL copied from the tutorial text in your browser.
```
I believe `URL` is correct way to type it according to the IEEE and general www documents?

Reply:
I think you are correct. We should check that this is consistent everywhere.